src/intentManager.js

```diff
@@ -153,6 +154,33 @@ define([
         }
     }
 
+    function printTemplate(mug, options) {
+        var widget = widgets.media(mug, options),
+            uploadComplete = widget.handleUploadComplete,
+            field_regex = /{{\s*(.+)\s*}}/gm;
```

Comment:
Can you some examples of what you're trying to match here? I think it will not do what you expect:

```js
>>> re = /{{\s*(.+)\s*}}/gm
/{{\s*(.+)\s*}}/gm
>>> str = "{{ abc }} {{ def }}"
"{{ abc }} {{ def }}"
>>> re.exec(str)
["{{ abc }} {{ def }}", "abc }} {{ def "]
>>> re.exec(str)
null
>>> re.exec(str)
["{{ abc }} {{ def }}", "abc }} {{ def "]
```

A guess that may be closer to what you want: `/{{\s*([^}]+?)\s*}}/gm`

However, this still has problems. It will not allow a `}` character inside a `{{` ... `}}` delimited range, which I assume should be legal given the double delimiter. Making a parser that would allow that is not trivial (maybe not even possible?) with a single regular expression.

This seems like a good candidate to extract into a helper function and add some tests to verify it does what you expect.

Also, it might be safer to define this regexp inside `handleUploadComplete` because the `g` flag makes it stateful. If the while loop condition ever changed it might not reset the regex state every time.